guides/common/modules/proc_configuring-custom-alternate-content-sources.adoc

```diff
@@ -1,5 +1,9 @@
-[id="Configuring_Alternate_Content_Sources_{context}"]
-= Configuring Alternate Content Sources
+[id="Configuring_Custom_Alternate_Content_Sources_{context}"]
+= Configuring Custom Alternate Content Sources
+
+.Prerequisites
+* If the repository requires SSL authentication, import the SSL certificate and key to the Satellite.
```

Comment:
```suggestion
* If the repository requires SSL authentication, import the SSL certificate and key to the your {ProjectServer}.
```

I am unsure what's really required as content credential. Is is just the certificate? See https://docs.theforeman.org/nightly/Managing_Content/index-katello.html#Importing_Custom_SSL_Certificates_content-management

Reply:
@maximiliankolb let me check with @ianballou once.

Reply:
@ianballou could you help with this?

Reply:
I'm not really understanding the exact question, but I'll lay it out:

SSL client cert, SSL client key, and SSL CA cert are set via content credentials.

If the upstream repo requires certificate authentication, use a client cert and key.

If you verify SSL, you can optionally include a CA certificate (again, as a content credential). If you have the upstream CA in your CA trust store, no need to include a CA certificate.

Reply:
Are we talking about something else than "content credentials within Foreman", which, to my understanding, always only contain the SSL certitficate as documented? Are we talking about adding things manually to the trust store of Foreman Server?

Reply:
It's just Foreman Content Credentials like you say, which can be ssl client certs, ssl client keys, or ssl ca certs (anything SSL Certificate related).

We don't need to mention adding anything to Foreman's trust store since the user can supply a CA certificate instead.